tasks/task264_paper_reviews_classification.json

```diff
@@ -0,0 +1,1038 @@
+{
+    "Contributors": "Mihir Parmar",
+    "Source": "Paper Reviews Dataset (dataset link- https://archive.ics.uci.edu/ml/datasets/Paper+Reviews)",
+    "Categories": "Classification",
+    "Definition": "In this task, you are given a set of paper reviews in english and spanish language. Based on given reviews, your job is to generate decision, i.e., \"accept\", \"reject\" or \"probably reject\" for the given paper.",
```

Comment:
Can we split this into two tasks? (one in Spanish and one in English).

Reply:
Dataset is taken from this paper (https://sentic.net/wisdom2017fuentes.pdf)

I think No. Because each paper has multiple reviews and some of them are English. So a set of a review sometimes contains both the language for reviews and it is not possible to apart them.

For example,

Input:

Review_1: I don't think the work reported in this paper is suitable for the SCCC conference. The author should consider reporting this experience on a conference more focused on practice in computer engineering than in CS reserarch. The material presented in the paper could be of interest for practicioners working with Oracle, namely a particular product. Review_2: Tema bien tratado técnicamente y en forma clara.  El tema solo muestra un manejo técnico sobre una herramienta, y no aporta algo novedoso. Se recomienda hacer una comparación en complejidad sobe las SQL con datos primitivos versus a datos XML, y abarcar otros aspectos como por ejemplo la mejora, etc. Review_3: Se ajusta al formato solicitado y se puede leer sin dificultad.  El artículo básicamente es un revisión de las herramientas Xpath, Xquery y SQL/XML en Oracle 10g, ilustrando su uso con varios ejemplos.  No hay un aporte, en ya sea nuevo conocimiento o aplicación de la tecnología para resolver un problema real.  No hay resultados de investigación que mostrar ni discutir.

(You can see this input contain both languages)

Output:
Accept (which is based on all three reviews)

So we cannot apart it.